### PR TITLE
Add Polish translation

### PR DESCRIPTION
**Pull request by mbiesiad**
Hi there! First of all, really great project. Polish translation of docs is [WIP]: https://github.com/mbiesiad/ava-docs/tree/pl_PL

 Stay tuned, best regards.

### CODE REVIEW

**Comment by novemberborn** [comment]
Wow, amazing effort! Thank you @mbiesiad! 

Can you think of anybody who could lightly review this?

Other than that please do leave a comment when this is ready.

**Comment by mbiesiad** [comment]
my pleasure! yup, sure. I will let you know, stay in touch

**Comment by mbiesiad** [comment]
@kekuu Could you take a look & CR?

**Comment by kekuu** [comment]
sure, looks OK

File: pl_PL/code-of-conduct.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+- [Najlepszym sposobem na odroczenie wyniku jest udowodnienie go nieudanym testem.](https://twitter.com/sindresorhus/status/579306280495357953)
+
+## Prześlij pull request
+
+- Nietrywialne zmiany należy najpierw omówić w kwestii: aby uniknąć wykonywania niepotrzebnej pracy.
+- W przypadku ważnych zadań radzimy jak najszybciej przedstawić swoje prace społeczności w celu uzyskania ich opinii. Otwórz pull request, gdy tylko zrobisz minimum niezbędne, aby zademonstrować swój pomysł. Na tym wczesnym etapie nie martw się, jeśli rzeczy nie są idealne lub w 100% skończone. Dodaj przedrostek [WIP] w tytule i opisz, co jeszcze musisz zrobić. Dzięki temu osoby patrzące na to nie szukają drobiazgów ani nie zgłaszają ulepszeń, które już wiesz, że musisz wprowadzić.
+- Nowym funkcjom muszą towarzyszyć testy i dokumentacja (w języku angielskim).
+- Nie dodawaj zmian niezwiązanych z pull requestem.
+- Sprawdź (Lint) i przetestuj przed wysłaniem pull requestem, uruchamiając `$ npm test`.
+- Złóż pull request [branche](https://github.com/dchelimsky/rspec/wiki/Topic-Branches), nie dla mastera.
```
Comment (kekuu) [inline pl_PL/code-of-conduct.md]: maybe better will be 'wyślij pull request'

**Review by kekuu — COMMENTED** [review]
LGTM

File: pl_PL/contributing.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+___
+# Wspieraj AVA
+
+✨ Dziękujemy za Twój wkład w AVA ! ✨
+
+Należy pamiętać, że ten projekt został opublikowany z [Code of conduct](code-of-conduct.md). Uczestnicząc w tym projekcie, zgadzasz się przestrzegać jego warunków.
+
+Tłumaczenia : [English](https://github.com/avajs/ava/blob/master/contributing.md), [Español](https://github.com/avajs/ava-docs/blob/master/es_ES/contributing.md), [Italiano](https://github.com/avajs/ava-docs/blob/master/it_IT/contributing.md), [日本語](https://github.com/avajs/ava-docs/blob/master/ja_JP/contributing.md), [Português](https://github.com/avajs/ava-docs/blob/master/pt_BR/contributing.md), [Русский](https://github.com/avajs/ava-docs/blob/master/ru_RU/contributing.md), [简体中文](https://github.com/avajs/ava-docs/blob/master/zh_CN/contributing.md)
```
Comment (kekuu) [inline pl_PL/contributing.md]: maybe better without white space -> "Tłumaczenia:"

File: pl_PL/contributing.md [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+
+## Jak mogę pomóc?
+
+### Popraw dokumentację
+
+Jako użytkownik AVA jesteś idealnym kandydatem, aby pomóc nam ulepszyć naszą dokumentację: poprawki literówek, poprawki błędów, lepsze wyjaśnienia, więcej przykładów itp. Otwarte problemy dotyczące rzeczy, które można poprawić. [Pomóż przetłumaczyć nasze dokumenty.](https://github.com/avajs/ava-docs) Cokolwiek, nawet ulepszenia wprowadzone w tym dokumencie.
+
+Użyj [etykiety `docs`](https://github.com/avajs/ava/labels/docs) aby znaleźć sugestie dotyczące tego, co chcielibyśmy zobaczyć więcej w dokumentacji.
```
Comment (kekuu) [inline pl_PL/contributing.md]: generally I think both correct co/czego, but maybe better sounds 'czego' here

File: pl_PL/contributing.md [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+- Napisz przekonujący opis, aby wyjaśnić, dlaczego powinniśmy zintegrować Twoje żądanie ściągnięcia. Twoim zadaniem jest nas przekonać. Odpowiedz na pytanie «dlaczego» jest wymagane i podaj przypadki użycia.
+- Możemy poprosić Cię o zmianę żądania ściągnięcia. Nie jest konieczne otwieranie kolejnego żądania ściągnięcia. [Po prostu zaktualizuj ten, który już istnieje.](https://github.com/RichardLitt/knowledge/blob/master/github/amending-a-commit-guide.md)
+
+Uwaga: przy modyfikowaniu kodu pamiętaj, że motto AVA (skradzione z Pythona) to jedna metoda zrobienia czegoś. Na przykład żądanie dodania aliasu do interfejsu API ([w ten sposób](https://github.com/avajs/ava/pull/663)) zostanie najprawdopodobniej odrzucone, jeśli nie zostaną dodane inne znaczące korzyści.
+
+*Czy chcesz wnieść swój pierwszy wkład w projekt typu open source? Nie szukaj dalej! AVA może być jednym z najbardziej przyjaznych projektów i społeczności. Przeczytaj post na blogu ["Making your first contribution"](https://medium.com/@vadimdemedes/making-your-first-contribution-de6576ddb190) aby zacząć poprawnie i zrobić swój pierwszy udział w AVA !*
+
+//Ten dokument tłumaczony został z języka francuskiego
```
Comment (kekuu) [inline pl_PL/contributing.md]: tłumaczony -> przetłumaczony

File: pl_PL/code-of-conduct.md (hunk 1 above)
Comment (mbiesiad) [reply pl_PL/code-of-conduct.md]: sure, agree! Thanks!

**Review by kekuu — APPROVED** [review]
LGTM

**Comment by novemberborn** [comment]
@mbiesiad thanks for all the activity here! @kekuu thanks for reviewing!

I take it this is good to go?

**Comment by mbiesiad** [comment]
@novemberborn please wait, a second. I will check again & I will let you know soon

**Comment by mbiesiad** [comment]
PS. By the way, I was even thinking about create an issue, but I will let you know here - the link for 
'JSPM and SystemJS' is unfortunately invalid - official docs (recipes): https://github.com/avajs/ava here: https://github.com/avajs/ava/blob/master/docs/recipes/jspm-systemjs.md

**Comment by novemberborn** [comment]
> I was even thinking about create an issue, but I will let you know here

Issue or PR welcome 😉

**Comment by mbiesiad** [comment]
> 
> 
> @mbiesiad thanks for all the activity here! @kekuu thanks for reviewing!
> 
> I take it this is good to go?

@novemberborn looks OK, yup.

**Comment by mbiesiad** [comment]
> 
> 
> > I was even thinking about create an issue, but I will let you know here
> 
> Issue or PR welcome 😉

done :-)

**Comment by novemberborn** [comment]
Awesome, thanks!

**Comment by mbiesiad** [comment]
my pleasure! Best wishes

**Pull request merged**